### ISSUE TITLE
Error Message

### ISSUE THREAD

**Issue by elliot-byford** [AwesomeTTS/awesometts-anki-addon#121]
Hi everyone, sorry if this isn't the appropriate place to be pasting error messages.

I'm using anki 2.1.29 (bbff62bf). When I click "insert" after trying to generate on-the-fly TTS, the following error message appears:

```
Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue on the add-on support site.
Debug info:
Anki 2.1.29 (bbff62bf) Python 3.8.0 Qt 5.15.0 PyQt 5.15.0
Platform: Mac 10.15.5
Flags: frz=True ao=True sv=1
Add-ons, last update check: 2020-08-19 12:18:32
Add-ons possibly involved: ⁨AwesomeTTS - Add speech to your flashcards⁩

Caught exception:
Traceback (most recent call last):
  File "/Users/Elliot/Library/Application Support/Anki2/addons21/1436550454/awesometts/gui/templater.py", line 205, in accept
    target = getattr(tform, now['templater_target'])
AttributeError: 'Ui_Form' object has no attribute 'back'
```

Any idea on how this can be fixed?

Thanks

**Comment by lucwastiaux** [comment]
same bug as #119

**Comment by lucwastiaux** [comment]
Please export your deck and share it with me, so I can reproduce this issue.

**Comment by mariuszmatusiak** [comment]
@lucwastiaux here is my deck. Thanks for your help. 
[[JP] JP -_ ENG_PL.zip](https://github.com/AwesomeTTS/awesometts-anki-addon/files/5135108/JP.JP.-_.ENG_PL.zip)

**Comment by lucwastiaux** [comment]
Hi @mariuszmatusiak I just tried to reproduce this issue using your deck and wasn't able to, would be able able to record a screen capture which shows the actions leading up to the crash ?
Thank you.

**Comment by elliot-byford** [comment]
Here is my deck, I hope it helps.

[Deck.apkg.zip](https://github.com/AwesomeTTS/awesometts-anki-addon/files/5141408/Deck.apkg.zip)

**Comment by elliot-byford** [comment]
> Please export your deck and share it with me, so I can reproduce this issue.

Here you go, I hope it helps!

[Deck.apkg.zip](https://github.com/AwesomeTTS/awesometts-anki-addon/files/5141411/Deck.apkg.zip)

**Comment by mariuszmatusiak** [comment]
@lucwastiaux here you go: 

[2020-08-29_10-15-35.zip](https://github.com/AwesomeTTS/awesometts-anki-addon/files/5144956/2020-08-29_10-15-35.zip)

**Comment by lucwastiaux** [comment]
@mariuszmatusiak thank you so much for your very detailed video, give me a little bit of time to analyze this.

**Comment by lucwastiaux** [comment]
OK i'm able to reproduce now, had to upgrade to Anki 2.1.32. Let me look into how to fix.

**Comment by lucwastiaux** [comment]
This seems to be due to a change in Anki 2.1.28: https://changes.ankiweb.net/#/?id=changes-in-2128

> Card template screen:
> 
>     Changes are now accumulated, and can be saved or discarded when you close the screen.
>     The front, back, and styling are no longer shown at the same time. You can switch between them with ctrl+1/2/3 or cmd+1/2/3.
>     Added a search bar to search for text in the template or styling.
>     Added a dropdown to change the previewed cloze number.
>     Added a checkbox to toggle the filling of empty fields for preview.
>     You can now delete a card template even if some notes are only using that template - they will be given a blank card 1 instead.

**Comment by lucwastiaux** [comment]
this will be fixed in 1.22, if you can't wait, grab the latest templater.py: https://github.com/AwesomeTTS/awesometts-anki-addon/blob/master/awesometts/gui/templater.py

**Comment by lucwastiaux** [comment]
@elliot-byford @mariuszmatusiak this has been fixed in AwesomeTTS 1.22. Please a review on the addon page if you're satisfied with the bugfix: https://ankiweb.net/shared/info/1436550454
